integration/spark/src/main/scala/org/apache/spark/sql/execution/command/mutation/merge/interfaces.scala

```diff
@@ -71,9 +72,11 @@ case class WhenNotMatchedAndExistsOnlyOnTarget(expression: Option[Column] = None
   override def getExp: Option[Column] = expression
 }
 
-case class UpdateAction(updateMap: Map[Column, Column]) extends MergeAction
+case class UpdateAction(var updateMap: Map[Column, Column], isStar: Boolean = false)
```

Comment:
wat does the `isStar` means here ?

Reply:
@ajantha-bhat it mean insert * or update *, * means or columns, this is desc in new added doc